cmd/commands/server.go

```diff
@@ -76,6 +78,7 @@ func NewServerCommand() *cobra.Command {
 	command.Flags().StringVar(&managedNamespace, "managed-namespace", sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_MANAGED_NAMESPACE", sharedutil.LookupEnvStringOr("NAMESPACE", "numaflow-system")), "The namespace that the server watches when \"--namespaced\" is \"true\".")
 	command.Flags().StringVar(&baseHref, "base-href", sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_BASE_HREF", "/"), "Base href for Numaflow server, defaults to '/'.")
 	command.Flags().BoolVar(&readOnly, "readonly", sharedutil.LookupEnvBoolOr("NUMAFLOW_SERVER_READONLY", false), "Whether to enable read only view for the UX server, defaults to false.")
+	command.Flags().BoolVar(&disableMetricsCharts, "disable-metrics-charts", sharedutil.LookupEnvBoolOr("NUMAFLOW_SERVER_DISABLE_METRICS_CHARTS", true), "Whether to disable metrics charts for the UX server, defaults to true.")
```

Comment:
Can we reuse the the configmap for this purpose? e.g. if the `url` is not configured, it means it's not enabled.

Reply:
we are passing this info to the UI via `/sysInfo` endpoint for which we create the object in `server/start.go`. We are defining the `PromQLService` inside `handler.go`. Should we move it outside at the top level?

Reply:
or similar to Dex Server, if we can move the `url` in shared config map, and use it to create the PromQLService object. wdyt?

Reply:
Use the 1st option?